src/appleseed.cli/main.cpp

```diff
@@ -267,7 +267,7 @@ namespace
         if (g_cl.m_passes.is_set())
         {
             params.insert_path(
-                "generic_frame_renderer.passes",
+                "passes",
```

Comment:
There was a bug here, no?

Reply:
No, because I need the frame to have access to this setting.